src/components/Projects/index.tsx

```diff
@@ -0,0 +1,190 @@
+import { Grid, Theme, useTheme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import PageHeaderWrapper from '../common/PageHeaderWrapper';
+import TextField from '../common/Textfield/Textfield';
+import useDebounce from 'src/utils/useDebounce';
+import { OrNodePayload, SearchRequestPayload } from 'src/types/Search';
+import { SearchService } from 'src/services';
+import { getRequestId, setRequestId } from 'src/utils/requestsId';
+import { useOrganization, useLocalization } from 'src/providers/hooks';
+import { Project } from 'src/types/Project';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  title: {
+    margin: 0,
+    fontSize: '24px',
+    fontWeight: 600,
+  },
+  contentContainer: {
+    backgroundColor: theme.palette.TwClrBg,
+    padding: theme.spacing(3),
+    borderRadius: '32px',
+  },
+  centered: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'flex-end',
+    marginBottom: '32px',
+  },
+  searchField: {
+    width: '300px',
+  },
+}));
+
+const columns = (): TableColumnType[] => [
+  { key: 'name', name: strings.NAME, type: 'string' },
+  { key: 'description', name: strings.DESCRIPTION, type: 'string' },
+];
+
+export default function ProjectsList(): JSX.Element {
+  const { selectedOrganization } = useOrganization();
+  const classes = useStyles();
+  const theme = useTheme();
+  const history = useHistory();
+  const [temporalSearchValue, setTemporalSearchValue] = useState('');
+  const debouncedSearchTerm = useDebounce(temporalSearchValue, 250);
+  const [results, setResults] = useState<Project[]>();
+  const { isMobile } = useDeviceInfo();
+  const contentRef = useRef(null);
+  const { activeLocale } = useLocalization();
+
+  const search = useCallback(
+    async (searchTerm: string, skipTfContact = false) => {
+      const searchField: OrNodePayload | null = searchTerm
+        ? {
+            operation: 'or',
+            children: [
+              { operation: 'field', field: 'name', type: 'Fuzzy', values: [searchTerm] },
+              { operation: 'field', field: 'description', type: 'Fuzzy', values: [searchTerm] },
+            ],
+          }
+        : null;
+
+      const params: SearchRequestPayload = {
+        prefix: 'projects',
+        fields: ['name', 'description', 'id', 'organization_id'],
+        search: {
+          operation: 'and',
+          children: [
+            {
+              operation: 'field',
+              field: 'organization_id',
+              type: 'Exact',
+              values: [selectedOrganization.id],
+            },
+          ],
+        },
+        sortOrder: [
+          {
+            field: 'name',
+          },
+        ],
+        count: 0,
+      };
+
+      if (searchField) {
+        params.search.children.push(searchField);
+      }
+
+      const searchResults = await SearchService.search(params);
```

Comment:
Do we want to use redux here, I understand it's a copy of existing code (which isn't the best to use as a template)... 
either move all this into projects service and just call the function with search term and org id, which returns back a list of Project[]?
Or use redux..?

Reply:
I agree it should be moved. I think in the past I've been putting these sorts of search queries into a service and using redux more for 1:1 entity APIs. Is that correct? What is the general pattern to deciding whether it goes into a service and is used in a component's state versus redux state?

Reply:
Redux was recently introduced into the codebase with the idea that all new functionality would 'ideally' use redux.
We didn't consider differentiating between using component state vs redux state for business data - maybe we should be open to that for sure.
As far as what goes into a service, anything involving wrangling types and parsing responses into types, etc. Search is a very good use-case, not sure components need to be aware search request payload semantics.. good candidate for services?

Reply:
@constanzauanini , at a minimum move the 'payload creation', search call and 'response parsing' into a service function. Should be good to go after that.

Reply:
Yes, I just copied existing code that was part of a component. But it should be part of a service. Will move it